examples/js/controls/OrbitControls.js

```diff
@@ -429,32 +429,24 @@ THREE.OrbitControls = function ( object, domElement ) {
 
 	function handleMouseDownRotate( event ) {
 
-		//console.log( 'handleMouseDownRotate' );
```

Comment:
If you want to propose changes unrelated to the topic of this PR, then please do so in a separate PR. ~12 instances. Thanks!